app/models/ability.rb

```diff
@@ -244,6 +245,7 @@ def public_ability
     can :index, Timeslot
     can :index, GogovanTransport
     can :index, CrossroadsTransport
+    can [:create, :show], OrganisationsUser
```

Comment:
This is a problem. We are allowing any non-logged in user to add any user to any organisation. We will need to fix this before deploying.

Reply:
Same case as mentioned below :( We can discuss this to clarify confusion and find better approach for this.

Reply:
There is no `show` action in the controller so we can remove it here.

Reply:
Agree. 👍